Jenkinsfile

```diff
@@ -125,6 +125,7 @@ pipeline {
             post {
                 always {
                     warnings consoleParsers: [[parserName: 'CppLint']], canRunOnFailed: true
+                    warnings consoleParsers: [[parserName: 'math-dependencies']], canRunOnFailed: true
```

Comment:
The `make test-math-dependencies` outputs things in the exact format as CppLint. Should we just use that as the parser? I don't know if the `parserName` is just a name or if it refers to a particular type of parser.

Reply:
This is leftover from before I was here. I don't know how these custom parsers work and messing with it is a separate issue.

Reply:
If this is the case, is this change necessary? Is there even a separate parser called `math-dependencies`?

Reply:
There is, yeah.